### DISCUSSION TITLE
save output in variable

### DISCUSSION THREAD

**Discussion by nareto** [casey/just#1126] [General]
Is it possible to grab the output of a command like in bash and use it afterwards? This 

```
cmd:
    out=$(ls /)
    echo "files: " $out
```

fails with `sh: 1: out: parameter not set`

**Comment by runeimp** [comment]
You need to setup the recipe in Shebang mode

```make
cmd:
	#!/bin/sh
	out=$(ls /)
	echo "files: " $out
```

or make it a one-liner

```make
cmd:
	out=$(ls /); echo "files: " $out
```

**Reply by nareto** [reply]
thanks! I knew I was missing something trivial
